app/models/experimentation/notifications_experiment.rb

```diff
@@ -42,34 +42,59 @@ def self.eligible_patients
                                              "end_time > ? OR treatment_group_memberships.created_at > ?",
                                              Time.current, MONITORING_BUFFER.ago
                                            ).select(:patient_id))
-        .then { |patients| exclude_bangladesh_blocks(patients) }
```

Comment:
I've deleted config-based filtering of India & BD environments. Post deployment, I'll update the active experiments in India's database with filters.
Bangladesh does not have any active experiment at the moment.

Reply:
Sounds good. Between deployment and updating the filter, there's no chance of an experiment running no?

Reply:
The enrollment happens at 5.45 AM, hence no.